src/js/view/controls/rightclick.js

```diff
@@ -56,16 +56,9 @@ export default class RightClick {
     }
 
     getOffset(evt) {
-        var target = evt.target;
-        // offsetX is from the W3C standard, layerX is how Firefox does it
-        var x = evt.offsetX || evt.layerX;
-        var y = evt.offsetY || evt.layerY;
-        while (target !== this.playerElement) {
-            x += target.offsetLeft;
-            y += target.offsetTop;
-
-            target = target.parentNode;
-        }
+        var playerBounds = bounds(this.playerElement);
+        var x = evt.clientX - playerBounds.left;
```

Comment:
This doesn't work once you're scrolled down on the page. Try `pageX` and `pageY`.